.github/workflows/contributioncheck.yml

```diff
@@ -0,0 +1,30 @@
+name: PR Contribution Rules Check
```

Comment:
will fail if issue reference not added. test by adding the said requirements and should run fine let me know otherwise